core/tests/protractor_utils/forms.js

```diff
@@ -280,11 +280,11 @@ var MultiSelectEditor = function(elem) {
         filteredElements[i].click();
         // Reopen the dropdown menu, since it closes after an item is
         // toggled.
-        elem.element(by.css('.dropdown-toggle')).click();
+        //elem.element(by.css('.uib-dropdown-toggle')).click();
```

Comment:
Delete this fully, don't just comment it out. (I assume this is an upgrade in ui-bootstrap's behaviour? Probably the comment above it can be removed too?)

Reply:
oops.. I meant to remove it.. missed that

Reply:
Done